### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by vanshitalavange**
Completed Alerts
image

### CODE REVIEW

File: stylesheets/base/colors.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+:root {
+  --primary-color: hsl(235, 86%, 65%);
+  --primary-hover: hsl(235, 87%, 70%);
+  --secondary-color: hsl(248, 10%, 30%);
+  --secondary-color:hsl(248, 10%, 40%);
```
Comment (ttahm3d) [inline stylesheets/base/colors.css]: I think this should have been --secondary-hover instead of --secondary-color.

**Review by ttahm3d — COMMENTED** [review]
Good work. Didn't find anything that needs to be changed apart from that secondary color hover name

File: stylesheets/base/colors.css (hunk 1 above)
Comment (vanshitalavange) [reply stylesheets/base/colors.css]: Oh yes, it was supposed to be that. Thank you for pointing out

**Review by tanishq20 — COMMENTED** [review]
Everything looks fine.
Good to merge

**Pull request merged**